src/components/Strategies/StrategyList.tsx

```diff
@@ -1,59 +1,39 @@
 import { Card, Row, Col, Typography } from 'antd';
 import styled from 'styled-components';
 
+import { IStrategies } from './../../types';
+
+import { strategies } from './StrategyData';
+
 const { Meta } = Card;
 
 const { Text } = Typography;
 
 function StrategyList() {
+  const strategyClick = (strategy: IStrategies) => {
+    var comingSoonTextId = document.getElementById(strategy.title);
+    if (strategy.displayComingSoonText) {
+      comingSoonTextId!.style.display = 'block';
```

Comment:
It's generally not a good practice to manipulate styles directly. It's better to apply classes instead. Define a new sub class, e.g. `.visible` and do `opacity: 0` by default and `opacity: 1` with the class applied